### ISSUE TITLE
PYTHON_EXE-NOTFOUND when python is in PATH

### ISSUE THREAD

**Issue by httpdigest** [google/shaderc#152]
OS: Windows 7 x64
Compiler: MSVC 14 "2015" x64
CMake: 3.5.0-rc3
Python: 2.7.11

There seem to be currently two different CMake properties/defines for where the Python exe is located.
My generated CMakeCache.txt looks like this:
```
//Path to a program.
PYTHON_EXE:FILEPATH=PYTHON_EXE-NOTFOUND

//Path to a program.
PYTHON_EXECUTABLE:FILEPATH=C:/Programme/Python27/python.exe
```
The CMake `-G "Visual Studio 14 2015 Win64"` runs successfully and also finds Python.
It seems that the vcxproj files only use the above property `PYTHON_EXE:FILEPATH` when they should've used the other one.

This causes the build to fail.
When I patch the above property in all vcxproj files to use the found python path, the whole build works and everything is fine.

**Comment by AWoloszyn** [comment]
Thanks for the report. I see what is wrong, and should have a fix up very soon.

**Comment by AWoloszyn** [comment]
This should be fixed by #154. There is a related change in spirv-tools that will be up soon.
